### PR TITLE
[SD-3896]: Added field management on content profiles

### PR DESCRIPTION
**Pull request by gbbr**
screen shot 2016-06-16 at 4 12 33 pm

### CODE REVIEW

File: scripts/superdesk-workspace/content/views/schema-editor.html [hunk 1]
```diff
@@ -17,7 +17,33 @@ <h4 translate>Schema Configuration</h4>
             </div>
 
             <div class="body" ng-if="!!props">
-                Required: <div sd-check ng-model="props.required"></div>
+                <form>
+                    <fieldset>
+                       <div class="field">
+                           Required: <div sd-check ng-model="props.required"></div>
```
Comment (petrjasek) [inline scripts/superdesk-workspace/content/views/schema-editor.html]: all those labels should be translated

**Comment by petrjasek** [comment]
looks good, but there is missing translate annotation for labels

**Comment by gbbr** [comment]
@petrjasek I've added `span[translate]`'s around those labels - is that what you were thinking? Also, there are the placeholders - do you reckon we should improve/extend (not sure if possible) the `translate` directive within this PR to include those?

**Comment by petrjasek** [comment]
:+1: yep that's it

**Pull request merged**